pandas/io/parquet.py

```diff
@@ -103,19 +103,27 @@ def __init__(self):
         self.api = pyarrow
 
     def write(self, df, path, compression='snappy',
-              coerce_timestamps='ms', **kwargs):
+              coerce_timestamps='ms', index=None, **kwargs):
         self.validate_dataframe(df)
-        if self._pyarrow_lt_070:
+
+        # Only validate the index if we're writing it.
+        if self._pyarrow_lt_070 and index is not False:
             self._validate_write_lt_070(df)
         path, _, _, _ = get_filepath_or_buffer(path, mode='wb')
 
+        if index is None:
+            from_pandas_kwargs = {}
+        else:
```

Comment:
can u always pass this? and just set index= if it’s None or does that change things?

Reply:
I am not sure if `preserve_index=None` is a valid value for pyarrow (at least not according to the docstring)

Reply:
I'd rather avoid messing with pyarrow like that, it's fairly finicky in my experience.